scripts/template_matcher.py

```diff
@@ -59,15 +60,63 @@ def _compute_prediction(self, k, img, kp, des):
         kp: keypoints from scene image,   des: descriptors from scene image
         """
 
-        # TODO: find corresponding points in the input image and the templae image
-        #       put keypoints from template image in template_pts
-        #       put corresponding keypoints from input image in img_pts
+        # http://docs.opencv.org/3.0-beta/doc/py_tutorials/py_feature2d/py_matcher/py_matcher.html
```

Comment:
Good citing of tutorials, and apologies that we didn't link to these tutorials earlier or give more guidance on how to do matching.